src/App.js

```diff
@@ -66,6 +73,137 @@ class App extends Component {
     });
   }
 
+  getLeaderboardHead() {
+    const w = parseInt(window.innerWidth, 10);
+
+    if (w <= 768) {
+      return (
```

Comment:
```js
let elem = null;
...
if (w <= 768) {
  elem = (
    <tr>
    ...
   )
}
```
The same with the else

Reply:
And add `return elem` in the end of method

Reply:
Done.